nomad/csi_endpoint.go

```diff
@@ -498,72 +545,3 @@ func (v *CSIPlugin) Get(args *structs.CSIPluginGetRequest, reply *structs.CSIPlu
 		}}
 	return v.srv.blockingRPC(&opts)
 }
-
-// controllerPublishVolume sends publish request to the CSI controller
-// plugin associated with a volume, if any.
-func (srv *Server) controllerPublishVolume(req *structs.CSIVolumeClaimRequest, resp *structs.CSIVolumeClaimResponse) error {
```

Comment:
I dunno why we originally had this sitting in `Server` instead of `ClientCSI`, but it doesn't need to be exposed at that level anymore. Moved it up with the rest of the methods for `ClientCSI`